metis-validation/metis-validation-service/src/main/java/eu/europeana/validation/service/Validator.java

```diff
@@ -120,15 +120,9 @@ private ValidationResult validate() {
         DOMResult result = new DOMResult();
         transformer.transform(new DOMSource(doc), result);
         NodeList nresults = result.getNode().getFirstChild().getChildNodes();
-        for (int i = 0; i < nresults.getLength(); i++) {
-          Node nresult = nresults.item(i);
-          if ("failed-assert".equals(nresult.getLocalName())) {
-            String nodeId = nresult.getAttributes().getNamedItem(NODE_ID_ATTR) == null ? null
-                : nresult.getAttributes().getNamedItem(NODE_ID_ATTR).getTextContent();
-            return constructValidationError(document,
-                "Schematron error: " + nresult.getTextContent().trim(), nodeId);
-          }
-        }
+        final ValidationResult validationResult = checkNodeListForErrors(nresults);
+        if (validationResult != null)
```

Comment:
Should use curly braces { } for this if statement.

Reply:
done